tests/Unit/FluidXPlate/Scalars/FluidXBarcodeTest.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Mll\LiquidHandlingRobotics\Tests\Unit\FluidXPlate\Scalars;
+
+use GraphQL\Error\Error;
+use GraphQL\Error\InvariantViolation;
+use Mll\LiquidHandlingRobotics\FluidXPlate\Scalars\FluidXBarcode;
+use PHPUnit\Framework\TestCase;
+
+class FluidXBarcodeTest extends TestCase
+{
+    public function testSerializeThrowsIfIsInvalid(): void
+    {
+        $this->expectException(InvariantViolation::class);
+        $this->expectExceptionMessage('The given value "foo123445" did not match the regex /[A-Z]{2}(\d){8}/.');
```

Comment:
```suggestion
        $this->expectExceptionObject(new InvariantViolation('The given value "foo123445" did not match the regex /[A-Z]{2}(\d){8}/.'));
```

Just a suggestion, the other way is fine too.